typescript-effect/src/lib.ts

```diff
@@ -52,7 +31,7 @@ export function program(port: number) {
             yield* scenario1(port),
             yield* scenario2(port),
             yield* scenario3(port),
-            yield* scenario4(port),
+            yield* scenario4(port).pipe(Effect.delay("10 seconds")),
```

Comment:
Why is this necessary?

Reply:
It is an alternative to dealing with this: https://github.com/jamesward/easyracer/blob/main/javascript-stdlib/lib.js#L62-L66